tools/run_tests/helper_scripts/build_python.sh

```diff
@@ -56,6 +56,12 @@ function is_linux() {
   fi
 }
 
+function inside-venv() {
```

Comment:
[Snake case is preferred for shell.](https://google.github.io/styleguide/shell.xml?showone=Function_Names#Function_Names)